front/lib/resources/subscription_resource.ts

```diff
@@ -182,4 +603,100 @@ export class SubscriptionResource extends BaseResource<Subscription> {
       requestCancelAt: null,
     };
   }
+
+  private static async isStripeSubscriptionOnProPlan(
+    owner: WorkspaceType,
```

Comment:
`nit`
```suggestion
    owner: LightWorkspaceType,
```

We try to use `LightWorkspaceType` everywhere unless we need the `ssoEnforced` which requires one more SQL query, we would use `WorkspaceType`.